src/main/java/de/tum/cit/aet/artemis/programming/service/ProgrammingExerciseRepositoryService.java

```diff
@@ -311,26 +308,25 @@ private void removeAuxiliaryRepository(AuxiliaryRepository auxiliaryRepository)
      * @throws IOException     Thrown in case resources could be copied into the local repository.
      * @throws GitAPIException Thrown in case pushing to the version control system failed.
      */
-    private void setupTemplateAndPush(final RepositoryResources repositoryResources, final String templateName, final ProgrammingExercise programmingExercise, final User user)
-            throws IOException, GitAPIException {
+    private void setupTemplateAndPush(RepositoryResources resources, String templateName, ProgrammingExercise programmingExercise, User user) throws IOException, GitAPIException {
```

Comment:
FYI: I wanted to keep the method signature in one line, `final` and long variable names are not really necessary